ui/org.eclipse.pde.core/src/org/eclipse/pde/internal/core/target/IUBundleContainer.java

```diff
@@ -598,6 +615,20 @@ public boolean getIncludeConfigurePhase() {
 		return fSynchronizer.getIncludeConfigurePhase();
 	}
 
+	/**
+	 * Returns whether or not repository references should be considered
+	 *
+	 * @return whether or not repository references should be considered
+	 */
+	public boolean getIncludeReferences() {
+		// if this container has not been associated with a container, return
+		// its own value
+		if (fSynchronizer == null) {
+			return (fFlags & INCLUDE_REFERENCES) == INCLUDE_REFERENCES;
```

Comment:
I think this is the usual pattern and avoids using the constant twice.

```suggestion
			return (fFlags & INCLUDE_REFERENCES) != 0;
```